.github/workflows/operator-verify.yaml

```diff
@@ -68,18 +68,20 @@ jobs:
       env:
         GARDENER_SA: ${{ secrets.GARDENER_SA }}
     - name: provision gardener
-      run: make -f hack/tools/gardener/Makefile provision-gardener
+      run: make -f hack/gardener.mk provision-gardener
```

Comment:
Why you use gardener.mk instead of Makefile? Makefile import gardener.mk and I remember that we discuss that .mk files shouldn't be used directly.

The same for all changes in this commit.